docs/Node/ReadLine.md

```diff
@@ -50,33 +66,33 @@ Prompt the user for input on the specified `Interface`.
 #### `setPrompt`
 
 ``` purescript
-setPrompt :: forall eff. String -> Int -> Interface -> Eff (console :: CONSOLE | eff) Interface
+setPrompt :: forall eff. String -> Int -> Interface -> Eff eff Interface
```

Comment:
Sorry, I don't think that's right. I don't think `noCompleter` is relevant here. If we go for the new effect route, it probably would make sense to add that to `Completer`. Maybe you could see how well it works with or without this effect by consulting the type checker? ;)